### PR TITLE
Vote/bar

### PR DESCRIPTION
**Pull request by noahnovickf**

### CODE REVIEW

File: src/containers/post/post.component.jsx [hunk 1]
```diff
@@ -1,29 +1,70 @@
...
+    if (obj.optionName === "option_a") {
+      setVoteACount(voteACount + 1);
+    } else {
+      setVoteBCount(voteBCount + 1);
+    }
+  };
+
+  const voteAPercent = (voteACount / (voteACount + voteBCount)) * 100;
+  const voteBPercent = (voteBCount / (voteACount + voteBCount)) * 100;
+  console.log(voteBPercent);
```
Comment (lolcode8) [inline src/containers/post/post.component.jsx:23]: Cleanup

File: src/store/thunks/users.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ export const signInWithGoogle = () => (dispatch) => {
           posts: [],
           full_name: res.user.displayName,
           avatar_link: "",
+          id: res.user.uid,
```
Comment (lolcode8) [inline src/store/thunks/users.js:18]: Can we call this `userId` instead of `id`?

Comment (noahnovickf) [reply src/store/thunks/users.js:18]: definitely

File: src/containers/post/post.component.jsx (hunk 1 above)
Comment (noahnovickf) [reply src/containers/post/post.component.jsx]: done

File: src/store/thunks/users.js (hunk 2 above)
Comment (noahnovickf) [reply src/store/thunks/users.js]: done

**Comment by noahnovickf** [comment]
Should be ready

**Pull request merged**